pkg/registry/apis/secret/contracts/outbox_queue.go

```diff
@@ -6,15 +6,32 @@ import (
 	secretv0alpha1 "github.com/grafana/grafana/pkg/apis/secret/v0alpha1"
 )
 
+var contextRequestIdKey = &struct{}{}
```

Comment:
I think the "idiomatic" way to do this is:
```suggestion
type contextRequestIdKey struct{}
```